addons/web/static/src/js/chrome/action_manager.js

```diff
@@ -718,8 +718,13 @@ var ActionManager = Widget.extend({
             });
         }
 
+        var self = this;
         return $.when(this[type](action, options)).then(function (executor_action) {
             options.on_load(executor_action);
+            // Display rainbowman on appropriate actions
+            if (action.effect) {
+                self.trigger_up('show_effect', action.effect);
+            }
```

Comment:
In which scenario is this useful, i mean, a scenario in which the previous code wasn't enough)?

Reply:
I just pushed a commit on this branch with an example that you can see

Reply:
Thanks but i can't find an example where [this](https://github.com/odoo/odoo/blob/master/addons/web/static/src/js/chrome/view_manager.js#L604}) isn't enough. This code is executed when a button is clicked, which is your usecase. A model's method is called, and it returns an action. The action has an awesome effect, which should thus be correctly displayed.

I am probably missing something...

Reply:
After discussing about it, we keep this fix for now